tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md

```diff
@@ -0,0 +1,34 @@
+Test 19-8 VIC SSO
+=======
+
+# Purpose:
+To verify that the SSO feature of VIC product works across Admiral, Harbor, and Engine
+
+# References:
+[1- VIC SSO Feature](TBD - waiting on official docs to link)
+
+# Environment:
+This test requires that a vCenter server with integrated PSC and with a discrete PSC is running and available
+
+# Test Steps:
+1. Download and install the VIC OVA from here:
+[OVA Releases](https://console.cloud.google.com/storage/browser/vic-product-ova-releases/?project=eminent-nation-87317&authuser=1)
+2. Navigate to the file server page https://<VIC>:9443
+3. Login into the vCenter using incorrect credentials
+4. Navigate to the admiral, harbor, demo VCH pages
+5. Login into the vCenter using correct credentials
+6. Navigate to the admiral, harbor, demo VCH pages
+7. Wait 10 minutes
+8. Navigate to the admiral, harbor, demo VCH pages
+9. Wait another 50 minutes (60 minutes total)
+10. Navigate to the admiral, harbor, demo VCH pages
+
+# Expected Outcome:
+* Steps 1,2,4-8 should all succeed without error
```

Comment:
Should this be `5-8` instead of `4-8`? Step 4 should fail upon entering incorrect creds - Harbor and Admiral are only started after correct creds have been entered.